src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -137,7 +157,6 @@
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
     <Reference Include="System" />
-    <Reference Include="System.Configuration" />
     <Reference Include="System.Core" />
     <Reference Include="System.Data" />
     <Reference Include="System.Drawing" />
```

Comment:
these assembly references (PresentationCore to System.Drawing) are only useful for net48. Maybe we can move them under the net48 ItemGroup.

Reply:
Let's clean up some of these References for net6 when we can.